### ISSUE TITLE
Webmentions not showing up as comments in 2024 theme on latest version

### ISSUE THREAD

**Issue by connectedblue** [pfefferle/wordpress-webmention#473]
I have a webmention on the following post which displays on version 5.2.6:

https://thoughts.uncountable.uk/finite-echo/

However, when I upgrade the plugin to any later version, including the latest, the comment disappears.

I'm using the 2024 theme on gutenberg.  I did come across the following thread when researching this:

https://github.com/WordPress/gutenberg/issues/58366

It suggests that the plugin may be missing a `pre_get_comments` callback, or similar.

**Comment by pfefferle** [comment]
This should be fixed in the latest version. I will close this for now, please re-open if it still exists.

**Comment by connectedblue** [comment]
Could I re-open this please?

I just upgraded to 5.3.2 and the same issue exists - the webmentions are not displayed on the url above (or any of the others on my site)

I rolled back to 5.2.6 and they work again.

**Comment by pfefferle** [comment]
Have you removed the comment form/section from the design completely?

**Comment by connectedblue** [comment]
> Have you removed the comment form/section from the design completely?

Yes.  I don't allow comments on my site, but I do log webmentions.  So the comments widget on this page:  https://thoughts.uncountable.uk/finite-echo/ displays webmentions received for this page (on version 5.2.6).  

When I upgrade the plugin to the latest version, that comment/webmention section disappears.

**Comment by pfefferle** [comment]
So you have the comments widget but not the form widget?

**Comment by connectedblue** [comment]
> So you have the comments widget but not the form widget?

Yes correct ... I edited the WP 2024 post template to remove the comment form but retained the comment widget

**Comment by pfefferle** [comment]
that makes it complicated 🤔
